app/components/topic-card/template.hbs

```diff
@@ -4,11 +4,34 @@
   {{/if}}
   {{topic.description}}
 </div>
-<button class='toggle-button-counter' disabled={{session.notAuthenticated}} {{action 'upvote'}}>
-  <div class='icon upvote {{if isUpvoter 'active'}}'></div>
-  <div class='count'>{{topic.upvoteCount}}</div>
-</button>
-<button class='toggle-button-counter' disabled={{session.notAuthenticated}} {{action 'volunteer'}}>
-  <div class='icon volunteer {{if isVolunteer 'active'}}'></div>
-  <div class='count'>{{topic.volunteerCount}}</div>
-</button>
\ No newline at end of file
+
+{{#if topic.talkBy}}
+  <div class="volunteer-container">
+    <img class='avatar' alt="{{topic.talkBy.username}}" title="{{topic.talkBy.username}}" src='{{topic.talkBy.avatar}}' />
+  </div>
+   {{date-format topic.topicDate format="MMMM Do"}}
+   {{#if talkByCurrentUser}}
+     <button class="link-btn" {{action "cancelTalkBy"}}>
+      [{{fa-icon 'times'}}]
+     </button>
+    {{/if}}
+{{else}}
+  <button class='toggle-button-counter' disabled={{session.notAuthenticated}} {{action 'upvote'}}>
+    <div class='icon upvote {{if isUpvoter 'active'}}'></div>
+    <div class='count'>{{topic.upvoteCount}}</div>
+  </button>
+  <button class="icon-btn" alt="Claim topic" title="Claim topic" {{action "chooseTalkDate"}} disabled={{session.notAuthenticated}}>
```

Comment:
It was a tricky one, if you looked at the ember inspector data panel the `talkDate` was null, but it was displaying today's date in the UI.  In the `date-format` helper you must have a `moment(value)` somewhere, and if value is `null` then that will just resolve to today. :wink: